regex-assembly/934140.ra

```diff
@@ -1,4 +1,4 @@
 ##! Please refer to the documentation at
 ##! https://coreruleset.org/docs/development/regex_assembly/.
 
-\@\{.*\}
+^(?:[^@]|@[^{])*@{.*}
```

Comment:
Please explain why this expression needs to be this complicated.
```suggestion
^(?:[^@]|@[^{])*@+{.*}
```

Reply:
Added.